common/dynamicconfig/constants.go

```diff
@@ -917,10 +915,9 @@ The API is under active development.`,
 
 	FrontendEnableUpdateWorkflowExecutionAsyncAccepted = NewNamespaceBoolSetting(
 		"frontend.enableUpdateWorkflowExecutionAsyncAccepted",
-		false,
+		true,
 		`FrontendEnableUpdateWorkflowExecutionAsyncAccepted enables the form of
-asynchronous workflow execution update that waits on the "Accepted"
-lifecycle stage. Default value is 'false'.`,
+asynchronous workflow execution update that waits on the "Accepted" lifecycle stage.`,
```

Comment:
Suggest ending all usage of "asynchronous" to mean the variant of Update that waits for Accepted. In https://docs.temporal.io/encyclopedia/workflow-message-passing#for-write-requests all Update usage is classified as synchronous. cc @drewhoskins-temporal